akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -342,6 +345,15 @@ object ShardRegion {
     var remaining = entities
 
     def receive = {
+      case ReceiveTimeout ⇒
+        log.warning("HandOffStopMessage[{}] is not handled by some of the entites of the `{}` shard, " +
+          "force stopping the remaining entities.", stopMessage.getClass.getName, shard)
```

Comment:
:+1:

Reply:
Typo: entities

Reply:
perhaps just stopping, as we're just using a regular stop.